R/upload.R

```diff
@@ -69,11 +69,18 @@ du.upload <- function(dict_version = "2_1", data_version = "1_0", dict_kind = du
       "Version: [ ", dict_version, " ] is not available in published data dictionaries. Possible dictionaries are: ",
       paste(ds_upload.globals$dictionaries_outcome, collapse = ", ")
     )
+  } else if (dict_version != "" && dict_kind == du.enum.dict.kind()$CHEMICALS && !(dict_version %in% ds_upload.globals$dictionaries_chemicals)) {
+    stop(
+      "Version: [ ", dict_version, " ] is not available in published data dictionaries. Possible dictionaries are: ",
+      paste(ds_upload.globals$dictionaries_chemicals, collapse = ", ")
+    )
   } else {
     if (dict_version == "" && dict_kind == du.enum.dict.kind()$CORE) {
       dict_version <- "2_1"
```

Comment:
This means the core variables of >2.1 don't work either, right? Interesting, we're supposed to be at 2.3 for core and 1.6 for outcome.

Reply:
If dict_version is empty the default version will be set to 2_1, you can set this to 1_0 or 2_0